src/components/ServerList.tsx

```
@@ -103,1052 +115,1073 @@ import { Skeleton } from "./ui/skeleton";
-      .fetchDataAndFilter()
-      .then(() => {
-        serverList.moveListDown();
-        let stringList: Array<{ server: string; motd: string }> = [];
-        let obj: any = {};
+	useEffectOnce(() => {
+		setRandomText(getRandomText());
+		serverList
+			.fetchDataAndFilter()
+			.then(() => {
+				serverList.moveListDown();
+				let stringList: Array<{ server: string; motd: string }> = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'stringList' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)